cas/build.gradle

```diff
@@ -74,11 +83,63 @@ task generateKeys {
   }
 }
 
-task runit {    
+task runit {
     doLast{
-        javaexec { 
+        javaexec {
             main="-jar";
-            args = [ "build/libs/cas.war" ]
+            jvmArgs = [
+                // Reduce the memory usage
+                "-Xmx2048M"
+                ]
+            args = [
+                "build/libs/cas.war",
+                "--logging.config=file:///home/cas/etc/cas/config/log4j2.xml",
+                "--spring.cloud.config.server.native.searchLocations=file:///home/cas/etc/cas/config"
```

Comment:
I don't understand why "/etc/cas/config" is default, would that not be easier to keep everything in the same folder where you do not have to have sudo permission on the server ? So that means to avoid the copyConfig and maybe use relative path to a folder "/cas/config" ?